src/mappers/json-to-journeyDetails.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Journeys,
+  Journey,
+  ApiResponseString,
+} from '../models/journeyResponse';
+
+export function jsonToJourneyDetail(json: ApiResponseString): Journeys {
+  return {
+    outboundJourneys: json.outboundJourneys.map(function (
+      outboundJourney,
+    ): Journey {
```

Comment:
I would generally use an arrow function for readability here - but doesn't matter too much